cumulus/client/cirrus-executor/src/tests.rs

```diff
@@ -133,8 +136,35 @@ async fn execution_proof_creation_and_verification_should_work() {
 		charlie.send_extrinsic(tx.clone()).await.expect("Failed to send extrinsic");
 	}
 
-	// Wait until the test txs are included in the next block.
-	charlie.wait_for_blocks(1).await;
+	// Ideally we just need to wait one block and the test txs should be all included
+	// in the next block, but the txs are probably unable to be included if the machine
+	// is overloaded, hence we manually mock the bundles processing to avoid the occasional
+	// test failure (https://github.com/subspace/subspace/runs/5663241460?check_suite_focus=true).
+	//
+	// charlie.wait_for_blocks(1).await;
+
+	let bundles = vec![OpaqueBundle {
+		header: BundleHeader {
+			primary_hash: alice.client.info().best_hash,
+			slot_number: Default::default(),
+			extrinsics_root: Default::default(),
+		},
+		opaque_extrinsics: test_txs
+			.iter()
+			.map(|xt| OpaqueExtrinsic::from_bytes(&xt.encode()).unwrap())
+			.collect(),
+	}];
+
+	charlie
+		.executor
+		.clone()
+		.process_bundles(
```

Comment:
This was a legacy method we wanted to remove, would be preferable to not add more uses of it and instead observe primary chain or something like that.

Reply:
Yeah, I'd love to but didn't find a nice way that just makes use of the native procedure but still ensures the test txs can be included in some block. It's indefinite if you have a really busy CPU from my local experiment.